README.rst

```diff
@@ -124,6 +124,45 @@ g. *Implementation Plan* (Optional): If accepted who will implement the change?
 
   This is also a good section to outline the changes to the implementation, or otherwise include insights that assume familiarity with the implementation.
 
+Here are some characteristics that a good proposal should have
+
+* It should be self-standing.  Some proposals accumulate a long and interesting discussion
+  thread, but in ten years time all that will be gone (except for the most assiduous readers).
+  Before acceptance, therefore, the proposal should be edited to reflect the fruits of
+  that dicussion, so that it can stand alone.
+
+* It should be precise, especially the "Proposed change specification"
+  section.  Language design is complicated, with lots of
+  interactions. It is not enough to offer a few suggestive examples
+  and hope that the reader can infer the rest.  Vague proposals waste
+  everyone's time; precision is highly valued.
+
+  We do not insist on a fully formal specification, with a
+  machine-checked proof.  There is no such baseline to work from, and
+  it would set the bar far too high.  On the other hand, for
+  proposals involving syntactic changes, it is very reasonable to ask for
+  a BNF for the changes.
+
+  Ultimately, the necessary degree of precision is a judgement that the committee
+  must make; but authors should try hard to offer precision.
+
+* It should offer evidence of utility.  Even the strongest proposals carry costs:
+
+  * For programmers: they usually make the language just a bit more complicated;
+  * For GHC maintainers: they make the implementation a bit more complicated;
+  * For those wanting to proposals in the future: they usually add new back-compat burdens, and consume syntactic design space.
+  * Moreover, all these costs contitute a permanent tax on every future programmer, langauge designer, and GHC maintainer.
```

Comment:
Maybe point out that removing extensions is much, much, harder than adding them?

Reply:
I have reworded slightly